### PR TITLE
[devtools] polish WNDT60

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

**Comment by kaycebasques** [comment]
Changed to P1, didn't mean to imply it needed to be out today

File: src/content/en/updates/2017/05/devtools-release-notes.md [hunk 1]
```diff
@@ -80,7 +129,7 @@ and how to contribute to it, check out the Lighthouse talk from Google I/O
 ### Third-party badges {: #badges }
 
 Use third-party badges to get more insight into the entities that are
-making network requests on a page and logging to the Console.
+making network requests on a page, logging to the Console, and executing JS.
```
Comment (jpmedley) [inline src/content/en/updates/2017/05/devtools-release-notes.md]: Spell out JavaScript.

File: src/content/en/updates/2017/05/devtools-release-notes.md [hunk 2]
```diff
@@ -134,6 +262,51 @@ highlights the jumpable destinations in blue.
...
+      "analytics": {
+        "label": label,
+        "value": 0
+      }
+    }
+  ]
+};
+</script>
+{% include "web/_shared/multichoice.html" %}
+{% endframebox %}
```
Comment (jpmedley) [inline src/content/en/updates/2017/05/devtools-release-notes.md]: This is the 3rd time I've seen this code. Can this be pulled into an external file and included where needed?

File: src/content/en/updates/2017/05/devtools-release-notes.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/updates/2017/05/devtools-release-notes.md]: Done

File: src/content/en/updates/2017/05/devtools-release-notes.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/updates/2017/05/devtools-release-notes.md]: Done

**Pull request merged**